frontend/src/views/web/challenge/submission.html

```diff
@@ -12,78 +12,114 @@
+                                    <input type="radio" name="selectPhase" class="with-gap selectPhase" id="{{item.id}}" value="{{item.id}}" ng-model="challenge.phaseId" ng-click="challenge.showRemainingSubmissions()">
+                                    <label for="{{item.id}}"></label>
+                                    <div class="show-member-title pointer"><strong class="text-med-black">Phase: </strong>{{item.name}}
+                                        <br>
+                                        <strong class="text-med-black">End Date: </strong>{{item.end_date | date:'medium'}}</div>
+                                    <div class="clearfix"></div>
+                                </li>
 
-                            <div class="row">
-                                <div class="file-field input-field col s6">
-                                    <div class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14">
-                                        <span>Upload File</span>
-                                        <input type="file" ngf-select ng-model="challenge.input_file" name="challenge.input_file" accept=".json">
-                                    </div>
-                                    <div class="file-path-wrapper">
-                                        <input class="file-path validate" type="text">
+                                <div class="row">
+                                    <div class="file-field input-field col s6" id="submission">
+                                        <div class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14">
+                                            <span>Upload File</span>
+                                            <input type="file" ngf-select ng-model="challenge.input_file" name="challenge.input_file" accept=".json">
+                                        </div>
+                                        <div class="file-path-wrapper">
+                                            <input class="file-path validate" type="text">
+                                        </div>
                                     </div>
                                 </div>
-                            </div>
 
-                            <div class="row">
-                                <div class="input-field align-left col s6">
-                                    <input type="text" id="methodName" name="methodName" ng-model="challenge.methodName" required>
-                                    <span class="form-icon"><i class="fa fa-text"></i></span>
-                                    <label for="methodName">Method Name *</label>
+                                <div class="row">
+                                    <div class="input-field align-left col s6" id="submission">
+                                        <input type="text" id="methodName" name="methodName" ng-model="challenge.methodName" required>
```

Comment:
Id should be unique @RishabhJain2018 , why you are uising one id at multiple places. Please use classes for this.